internal/report.go

```diff
@@ -439,12 +452,26 @@ func writeUnexpectedConditions(conv *Conv, w *bufio.Writer) {
 		reparseInfo()
 		return
 	}
-	w.WriteString("For debugging only. This section provides details of unexpected conditions\n")
-	w.WriteString("encountered as we processed the pg_dump data. In particular, the AST node\n")
-	w.WriteString("representation used by the lfittl/pg_query_go library used for parsing\n")
-	w.WriteString("pg_dump output is highly permissive: almost any construct can appear at\n")
-	w.WriteString("any node in the AST tree. The list details all unexpected nodes and\n")
-	w.WriteString("conditions.\n")
+	if driverName == "mysqldump" {
```

Comment:
Sure, I will put a `switch` statement instead of `if` confition  
But Enums are declared in `main.go` and we are sending an argument of `driverName` to `GenarteReport()` function. So, either we have to declare constants again in `report.go` or we would need to write the driverName as being currently done.

Let me know if you think there is some other way possible.

Reply:
I would prefer to have a central place (in the package) to manage all support drivers. Let's wait for @nevinheintze 's opinion on this.

Reply:
As discussed in the other thread on main.go, I kind of agree none of the options are really a big improvement.
Let's leave as is.
I'll resolve the thread.